components/snowflake/sources/new-schema/new-schema.mjs

```diff
@@ -0,0 +1,47 @@
+import common from "../common.mjs";
+import { v4 as uuidv4 } from "uuid";
+
+export default {
+  ...common,
+  type: "source",
+  key: "snowflake-new-schema",
+  name: "New Schema",
+  description: "Emit new event when a schema is created",
```

Comment:
```suggestion
  description: "Emit a new event when a schema is created",
```

Reply:
Changed "Emit new event" to "Emit a new event" to follow Google's Developer docs style guide (use articles)